doc/doxygen/src/MainPage.dox

```diff
@@ -1,5 +1,27 @@
 /*! @mainpage An Introduction
  *
+ * @note Looking for resources?
+ * * 🐍 Get OpenAssetIO from <a
+     href="https://pypi.org/project/openassetio/">PyPI</a>
+ * * 🛠️ Learn how to <a
+     href="https://github.com/OpenAssetIO/OpenAssetIO/blob/main/doc/BUILDING.md">build
+     OpenAssetIO</a>
+ * * 📐 Get off to a good start with the <a
+     href="https://github.com/OpenAssetIO/Template-OpenAssetIO-Manager-Python">python
```

Comment:
[consider][minor] Consistent captialisation
```suggestion
     href="https://github.com/OpenAssetIO/Template-OpenAssetIO-Manager-Python">Python
```
I wonder if we should capitalise some more, i.e. "Manager", "Media Linker"

Reply:
You know I agonised over this, wondering what you might think is correct. Since all the others are Proper-Nounish (ie they're project names,) i thought this one should be lowercase.

Nonetheless, took your suggestion here, but didn't change any others.